pnpm-workspace.yaml

```diff
@@ -7,4 +7,5 @@ packages:
   - infra
   - test
   - ui
-  - merkle-tree-wasm
+  - merkle-tree-wasm/*
+  - workers/*
```

Comment:
Yes, since `workers` already is a workspace it self that has only web-worker library packages. And for `merkle-tree-wasm` it has two packages, the Rust package it self and the wasm package too.